online-shop/src/app/confirm-dialog/confirm-dialog.component.scss

```diff
@@ -0,0 +1,5 @@
+.content-container {
+    width: 350px;
```

Comment:
as TIP: https://www.w3schools.com/cssref/css_units.php
the pixel density differs from one monitor to another, you will think the task is working fine, and then you will get a tester that will tell you that is not good :D